src/queries/get-product-slug.js

```diff
@@ -1,16 +1,15 @@
 import gql from 'graphql-tag';
```

Comment:
Now we can include gql from `@apollo/client` . can we have that in every query and remove gql-tag dep ?